src/Nox.ClientApi.Tests/Tests/Controllers/CountriesControllerTests.cs

```diff
@@ -153,6 +153,33 @@ public async Task Get_OwnedEntityByParentKeyAndFilter_ReturnsSingleEntity()
         }
         #endregion
 
+        #region GET Owned Entity via Parent Key /api/{EntityPluralName}/{EntityKey}/{OwnedEntityPluralName}/{OwnedEntityKey} => api/countries/1/CountryLocalNames/1
+        [Fact]
+        public async Task Get_OwnedEntityByParentKey_ReturnsOwnedEntity()
+        {
+            var expectedCountryLocalName = _fixture.Create<string>();
+            // Arrange
+            var dto = new CountryCreateDto
+            {
+                Name = _fixture.Create<string>(),
+                CountryLocalNames = new List<CountryLocalNameCreateDto>() {
+                    new CountryLocalNameCreateDto() { Name = expectedCountryLocalName }
+                }
+            };
+            var result = await _oDataFixture.PostAsync<CountryCreateDto, CountryDto>(EntityUrl, dto);
+            var country = await _oDataFixture.GetAsync<CountryDto>($"{EntityUrl}/{result!.Id}");
```

Comment:
we have new GetOdataSingleResponse for this,  you juts need to update the test

Reply:
Updated. Also I've added Skip for one test that is pending OData Action/Function implementation for non-conventional endpoints.